docker/scylla-sct/ubuntu/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN apt-get update && \
     adduser --disabled-password --gecos "" $USER || true  && \
     usermod -aG sudo $USER && \
     sudo -Hu $USER sh -c "mkdir -m 700 ~/.ssh" && \
+    sudo -Hu $USER sh -c "echo 'mkdir -p /home/$USER/.cassandra && ! test -f /home/$USER/.cassandra/cqlshrc && sudo cp /root/.cassandra/cqlshrc /home/$USER/.cassandra/cqlshrc' >> ~/.ssh/rc" && \
```

Comment:
Work locally, and in https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/artifacts-docker-test/19/

it's owned by root
```
scylla-test@6e8d27e237cf:~$ ls -all ~/.cassandra/
total 33
drwxrwxr-x 2 scylla-test scylla-test   5 Dec 26 12:31 .
drwxr-x--- 5 scylla-test scylla-test  10 Dec 26 12:31 ..
-rw------- 1 scylla-test scylla-test   5 Dec 26 12:31 cqlsh_history
-rw-r--r-- 1 root        root         35 Dec 26 12:30 cqlshrc
-rw-rw-r-- 1 scylla-test scylla-test 541 Dec 26 12:40 nodetool.history
```

but still cqlsh can read it, since the Dockerfile is doing `usermod -aG sudo $USER`

so this user has sudo permissions

Reply:
in general it would be nice if we have replaced the ssh with `docker exec`, similar to what we are doing in k8s
and then all that can be thrown out,

but that quite much work for 1 formal artifact docker job.